test/testdata/fix/out/whitespace.go

```diff
@@ -56,4 +56,20 @@ func multiIfFunc() {
 
 		fmt.Println("Hello multi-line world")
 	}
+
+	if true {
+		if true {
+			if true {
+				if 1 == 1 &&
+					2 == 2 {
+
+						fmt.Println("Hello nested multi-line world")
+				}
+			}
+		}
+	}
+}
+
+func notGoFmted() {
+         fmt.Println("Hello world")
```

Comment:
Noticed here that we have the same situation as in https://github.com/golangci/golangci-lint/pull/4003#discussion_r1290737071. Basically this results in the standalone linter and `golangci-lint` not fixing the same way, `golangci-lint` will leave the sapces before `fmt` whereas the linter itself will replace it with a tab.

I don't think there's anything we should do extra in this specific linter for this scenario (and also that's the same behaviour as before, we just didn't have a standalone linter) but yet another reason to try to integrate the suggested fixes with the `golangci-lint` fixer.